app/src/main/res/values/strings.xml

```diff
@@ -56,77 +55,77 @@
-    use them, for instance, to store user settings (e.g. search settings). But they could also be used to collect personal
+        use them, for instance, to store user settings (e.g. search settings) but they could also be used to collect personal
         user data. Third party cookies (they collect user data for advertising) are always disabled.</string>
     <string name="setting_summary_cookie_delete">If enabled, all cookies will be deleted, including cookies of trusted websites.</string>
     <string name="setting_title_whitelistCookie">Cookie whitelist</string>
-    <string name="setting_summary_whitelistCookie">If you disable Cookies, you can use this whitelist to allow sites you
+    <string name="setting_summary_whitelistCookie">If you disabled Cookies you can use this whitelist to allow sites you
         trust to use save cookies.</string>
 
     <string name="setting_title_javascript">JavaScript</string>
     <string name="setting_summary_javascript">Enable JavaScript. Many websites need JavaScript for their functionality (menus,
-        loading and displaying images, …). But they are also a risk and can be used for attacks.</string>
+        loading and displaying images and more) but it can also be used for attacks.</string>
     <string name="setting_title_whitelistJS">JavaScript whitelist</string>
-    <string name="setting_summary_whitelistJS">If you disable JavaScript, you can use this whitelist to allow sites you
+    <string name="setting_summary_whitelistJS">If you disabled JavaScript you can use this whitelist to allow sites you
         trust to use JavaScript.</string>
 
     <string name="setting_title_adblock">AdBlock</string>
-    <string name="setting_summary_adblock">Enable AdBlock to block annoying ads. Sites also load faster when enabled.</string>
+    <string name="setting_summary_adblock">Enable AdBlock to block annoying ads. Sites will load faster with it enabled.</string>
     <string name="setting_title_whitelist">AdBlock whitelist</string>
-    <string name="setting_summary_whitelist">Allow these sites to show ads. Some sites do not work when AdBlock is enabled.</string>
+    <string name="setting_summary_whitelist">Allow these sites to show ads. Some sites will not work if AdBlock is enabled.</string>
 
     <string name="setting_title_location">Location</string>
     <string name="setting_summary_location">Allow websites to access your location. Some websites (like Google Maps for navigation)
         need this enabled.</string>
 
     <string name="setting_title_history">History</string>
-    <string name="setting_summary_history">Save the history of visited websites.</string>
-    <string name="save_history_mode_save_when_open">Save when opening</string>
+    <string name="setting_summary_history">Save the history of visited websites</string>
+    <string name="save_history_mode_save_when_open">Save when opening</string> <!-- When opening tab or EinkBro? -->
```

Comment:
when opening a new tab, does EinkBro save that link to Tab List status right away (for next time launch), or save the  link until that tab is closed.

Reply:
Thanks for clarification!
What do you think about changing _"Save when opening / closing"_ 2 strings to something more self-explanatory?
I would change them to either _"Save on tab opening"_ or _"Save when opening tab"_ and the same for _"Save when closing"_.